src/manager/talent.manager.ts

```diff
@@ -39,15 +39,37 @@ export class TalentManager {
 	}
 
 	public async createTalent(body: any): Promise<Talent> {
-		const url: string = await this.gravityCloudService.upload(body.listingImage);
+		const promise1 = this.gravityCloudService.upload(body.listingImage);
+		const promise2 = this.gravityCloudService.upload(body.listingCroppedImage);
+		const promise3 = this.gravityCloudService.upload(body.profileImage);
+		const promise4 = this.gravityCloudService.upload(body.profileCroppedImage);
+		const values: Array<string> = await Promise.all([promise1, promise2, promise3, promise4]);
 
 		unlink(body.listingImage, (err) => {
```

Comment:
create a function that deletes a file and you can call it multiple times, like this:
deleteTempFile(body.listingImage);
deleteTempFile(body.profileImage);
deleteTempFile(body.listingImage);
deleteTempFile(body.profileImage);

or create a function that accepts a list of files and uses a for to delete them. but don't use 5 x number_of_files lines of code that do pretty much the same thing